# Found flag that takes on values 1 or 0 (cylinder detected or not)
uint8 found

# Cylinder location in map coordinates
float64 x
float64 y
float64 z